inorbit_edge/robot.py

```diff
@@ -469,6 +470,36 @@ def report_command_result(self, args, execution_id, result_code):
         msg.return_code = result_code
         self.publish_protobuf(MQTT_SCRIPT_OUTPUT_TOPIC, msg)
 
+    def register_executable_commands(
+        self, exec_name_regex=r".*\.sh", path="./user_scripts"
```

Comment:
nit: I'd reverse the order of path and regex. In fact, you could make regex optional and in that case ALL custom commands will be funneled to executed here, like the agent does.